jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java

```diff
@@ -40,10 +102,25 @@ public void setLength(double length) {
         this.length = length;
     }
 
+    /**
+     * Retrieves the speedup factor applied by the layer for this action.
+     * The speed factor gets applied on each interpolation by this formula : time += tpf * currentAction.getSpeed() * ac.globalSpeed.
```

Comment:
Can you change `ac.globalSpeed` to `composer.getGlobalSpeed()`. Also `currentAction.getSpeed()` can be simplified to `action.getSpeed()`.

after "The speed factor gets applied" would be nice to add "inside AnimLayer on each interpolation step..."

Reply:
Thanks for the suggestions!